defaults/validate.go

```diff
@@ -32,6 +38,14 @@ func Validate(value interface{}) error {
 		}
 	}
 
+	// For non-struct values, we cannot do much, as there's no associated tags
+	// to lookup to decide how to validate, so we have to assume they're valid.
+	if t.Kind() != reflect.Struct {
+		return nil
+	}
```

Comment:
This change is not directly related to the PR, but we need it to be able to validate slices of basic types, as the slice of strings (`PermalinkIPsToReplace`) this PR adds. It's actually the first time we add a slice of basic types in a config, and that's why this never failed.
The other comment in this file was added because it took me a lot of time to understand that piece of code :nerd_face: